### DISCUSSION TITLE
Support intermediary values automatically

### DISCUSSION THREAD

**Discussion by rentalhost** [tailwindlabs/tailwindcss#11171] [Ideas]
By default, some namespaces are limited by default setting and can be expanded via custom settings. For instance, `opacity-80` is default support, but `opacity-70` is not, but can be expanded:

```ts
module.exports = {
  theme: {
    // Replaces all of the default `opacity` values
    opacity: {
      // ...
      '70': '0.7',
    }
  }
}
```

The same happens to color.

It would be great if it were possible to support intermediate values natively, without explicitly using arbitrary values for some cases, mainly for colors.

`opacity-70` instead of `opacity-[0.7]`

And add color fragmentation support:

`text-red-650`, `text-red-625`

**Comment by wongjn** [comment]
[`opacity-70` already exists with the default configuration](https://tailwindcss.com/docs/opacity).

For more color shades, it could be a lot of work since [all the colors were picked out by hand](https://tailwindcss.com/docs/customizing-colors#generating-colors).

**Reply by rentalhost** [reply]
Ah, that was a bad example from me. haha I mean `opacity-55` for instance. 

While I understand that shades are picked out by hand, wouldn't it be possible to calculate a shade between 2 shades? For example, `red-550` would be an intermediate between `red-500` and `red-600`. And `blue-225` would be closer to `blue-200` than `blue-300`.

Currently, I need to do this manually for each color I need. I'm looking for a way to do this via a plugin, maybe it's easier and I don't need to impact TW directly. But it would be great if this feature could be native as well.
